### DISCUSSION TITLE
Rebroadcast Plugin Crashing

### DISCUSSION THREAD

**Discussion by RayRein** [koush/scrypted#1588] [Q&A]
I'm trying to set up a Reolink 810A POE camera using Home Assistant and Apple HomeKit. On both platforms, I can see LIVE video stream, but nothing is being recorded. (Motion-triggered events are being recorded on a Reolink DVR, which i can review using the Reolink web-based app.) This is really frustrating because the system worked correctly (including recording to Apple Secure Video) until I had to disconnect everything to move my camera. Any help would be greatly appreciated.

The problem appears to be with the Rebroadcast Plugin. This is the output of the camera log, the latter part of which keeps repeating:

[Rebroadcast Plugin]: prebuffer sessions starting in 5 seconds
[Rebroadcast Plugin]: RTMP main.bcs prebuffer session started
[Rebroadcast Plugin]: RTMP main.bcs prebuffer session starting
[Rebroadcast Plugin]: stream RTMP ext.bcs is not enabled and will be rebroadcast on demand.
[Rebroadcast Plugin]: stream RTMP sub.bcs is not enabled and will be rebroadcast on demand.
[Rebroadcast Plugin]: stream RTSP h264Preview_01_main is not enabled and will be rebroadcast on demand.
[Rebroadcast Plugin]: stream RTSP h264Preview_01_sub is not enabled and will be rebroadcast on demand.
[Rebroadcast Plugin]: -hide_banner -fflags +genpts -i rtmp:[REDACTED] -rtsp_transport tcp -vcodec copy -acodec copy -f rtsp rtsp://127.0.0.1:41967/b2f82658c2b21ad1
[Rebroadcast Plugin]: rtmp://192.168.1.141:1935/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=Rei727887: End of file
[Rebroadcast Plugin]: ffmpeg exited
[Rebroadcast Plugin]: Reolink Camera restarting prebuffer session in 5 seconds
[Rebroadcast Plugin]: RTMP main.bcs prebuffer session ended with error Error: ffmpeg exited
[Rebroadcast Plugin]:     at ChildProcess.eval (/@scrypted/prebuffer-mixin/main.nodejs.js:1:68418)
[Rebroadcast Plugin]:     at ChildProcess.emit (node:events:531:35)
[Rebroadcast Plugin]:     at ChildProcess.emit (node:domain:488:12)
[Rebroadcast Plugin]:     at ChildProcess._handle.onexit (node:internal/child_process:294:12)
[HomeKit]: HomeKit Connection 192.168.1.104
[Rebroadcast Plugin]: RTMP main.bcs prebuffer session started
[Rebroadcast Plugin]: RTMP main.bcs prebuffer session starting
[Rebroadcast Plugin]: -hide_banner -fflags +genpts -i rtmp:[REDACTED] -rtsp_transport tcp -vcodec copy -acodec copy -f rtsp rtsp://127.0.0.1:38915/dadcad48a3e6c57b

**Comment by RayRein** [comment]
Thought I was making progress, but then found that the main.bcs prebuffer is continually crashing. More output from log:

<img width="777" alt="image" src="https://github.com/user-attachments/assets/0dadf553-7e9a-408e-8cea-60834d00a7fb">

**Comment by mzamponi-dermalog** [comment]
you are aware that there is a password in the log you posted?
